src/main/scala/gestalt/Quote.scala

```diff
@@ -11,6 +11,7 @@ abstract class Quote(val t: Toolbox, val toolboxName: String) {
   type Quasi = m.internal.ast.Quasi
 
   // fields
+  // args: List[t.Tree[Any]]
```

Comment:
`List[t.Tree[Any| t.Tree[_]]]` is too much noise. I think `_` can express what you mean, as it means it can be instantiated by any type. `Any` is problematic to some extent, as there is indeed a type `Any`.